cli/azd/resources/scaffold/templates/host-containerapp.bicept

```diff
@@ -331,14 +368,14 @@ resource appLinkToPostgres 'Microsoft.Resources/deploymentScripts@2023-08-01' =
   properties: {
     azCliVersion: '2.63.0'
     timeout: 'PT10M'
-    scriptContent: 'apk update; apk add g++; apk add unixodbc-dev; az extension add --name containerapp; az extension add --name serviceconnector-passwordless --upgrade; az containerapp connection create postgres-flexible --connection appLinkToPostgres --source-id ${app.id} --target-id ${postgresDatabaseId} --client-type springBoot --user-identity client-id=${identity.properties.clientId} subs-id=${subscription().subscriptionId} user-object-id=${linkerCreatorIdentity.properties.principalId} -c main --yes; az tag create --resource-id ${app.id} --tags azd-service-name={{.Name}} '
+    scriptContent: 'apk update; apk add g++; apk add unixodbc-dev; az extension add --name containerapp; az extension add --name serviceconnector-passwordless --upgrade; az containerapp connection create mysql-flexible --connection appLinkToMySql --source-id ${app.id} --target-id ${mysqlDatabaseId} --client-type springBoot --user-identity client-id=${identity.properties.clientId} subs-id=${subscription().subscriptionId} user-object-id=${linkerCreatorIdentity.properties.principalId} mysql-identity-id=${mysqlIdentityName} -c main --yes;'
     cleanupPreference: 'OnSuccess'
     retentionInterval: 'P1D'
   }
 }
 {{- end}}
-
 {{- if (and .AzureServiceBus .AzureServiceBus.AuthUsingManagedIdentity) }}
+
```

Comment:
FYI: The white space will be kept in the generated bicep file only when the white space put after fragment like `{{-if`.